corehq/apps/app_manager/static/app_manager/js/details/column.js

```diff
@@ -45,19 +45,20 @@ hqDefine("app_manager/js/details/column", function () {
         self.original.case_tile_field = ko.utils.unwrapObservable(self.original.case_tile_field) || "";
         self.case_tile_field = ko.observable(self.original.case_tile_field);
 
-        // Set up tab attributes
+        // Set up tab defaults
         var tabDefaults = {
             isTab: false,
             hasNodeset: false,
             nodeset: "",
-            // Blank string is meaningful for nodesetCaseType (indicating a custom expression), so be strict here
-            nodesetCaseType: _.has(self.original, "nodesetCaseType") ? self.original.nodesetCaseType : screen.childCaseTypes[0] || "",
```

Comment:
This line is the bug: nodesetCaseType shouldn't default to `screen.childCaseTypes[0]` if there's a nodeset present. This logic is already a lot for one line, so I moved it down to happen after the rest of the defaulting.